scripts/utils/generate_ts.sh

```diff
@@ -21,82 +21,101 @@ mkdir -p cache || die
 cp scripts/utils/generate_strings.py cache || die
 print G "done."
 
-printn Y "Generating strings... "
-python cache/generate_strings.py translations/strings.yaml -o translations/generated
-print G "done."
+for project in src/apps/*; do
+  project=$(basename $project)
+
+  mkdir -p translations/generated/$project || die
 
-printn Y "Generating a dummy PRO file... "
-mkdir -p translations/generated || die
-cat > translations/generated/dummy_ts.pro << EOF
+  printn Y "$project - Generating strings... "
+  python3 cache/generate_strings.py src/apps/$project/translations/strings.yaml -o translations/generated/$project
+  print G "done."
+
+  printn Y "$project - Generating a dummy PRO file... "
+  cat > translations/generated/$project/dummy_ts.pro << EOF
 HEADERS += i18nstrings.h
 HEADERS += \$\$files(../../src/shared/*.h, true)
-HEADERS += \$\$files(../../src/apps/vpn/*.h, true)
+HEADERS += \$\$files(../../src/apps/$project/*.h, true)
 HEADERS += \$\$files(../../nebula/*.h, true)
 
 SOURCES += i18nstrings_p.cpp
 SOURCES += ../i18nstrings.cpp
```

Comment:
Why do we need two for this file?

Reply:
because `relay` and other "apps" will have their own folder.

Reply:
(I missed the names were slightly different, I thought it was the same file)